src/Loading/sceneLoader.ts

```diff
@@ -525,56 +514,46 @@ export class SceneLoader {
             });
         }
 
-        if (fileInfo.file) {
-            // Loading file from disk via input file or drag'n'drop
-            const errorCallback = (error: ReadFileError) => {
-                onError(error.message, error);
-            };
-
-            request = plugin.readFile
-                ? plugin.readFile(scene, fileInfo.file, dataCallback, onProgress, useArrayBuffer, errorCallback)
-                : scene._readFile(fileInfo.file, dataCallback, onProgress, useArrayBuffer, errorCallback);
-        } else {
-            const manifestChecked = () => {
-                if (pluginDisposed) {
-                    return;
-                }
-
-                const successCallback = (data: string | ArrayBuffer, request?: WebRequest) => {
-                    dataCallback(data, request ? request.responseURL : undefined);
-                };
+        const manifestChecked = () => {
+            if (pluginDisposed) {
+                return;
+            }
 
-                const errorCallback = (error: RequestFileError) => {
-                    onError(error.message, error);
-                };
+            const successCallback = (data: any, responseURL?: string) => {

```

Comment:
as this function is exactly the dataCallback function, is there a need to define it here?

Reply:
Good point. This isn't necessary anymore. We can just pass dataCallback directly.